lib/net.c

```diff
@@ -160,14 +160,12 @@ ethertype_filter_add(struct gatekeeper_if *iface, uint16_t ether_type,
  * By specifying the tuple (proto, src_port, dst_port) (and masks),
  * it can filter both IPv4 and IPv6 addresses.
  */
-int
-ntuple_filter_add(struct gatekeeper_if *iface, uint32_t dst_ip,
+static int
+ipv4_ntuple_filter_add(struct gatekeeper_if *iface, uint32_t dst_ip,
 	uint16_t src_port, uint16_t src_port_mask,
 	uint16_t dst_port, uint16_t dst_port_mask,
```

Comment:
1. Let's use `rte_be16_t` here too.
2. We can use `rte_be32_t` for `dst_ip`. This means doing the same in `ipv4_ntuple_filter_add` as well.